dips/dip-20.md

```diff
@@ -187,7 +191,7 @@ denomination coin in common usage is one cent (1/100th of an XUS dollar),
 so the `fractional_part` for XUS would be
 100. If you wanted to translate from an on-chain value `v` to a number that
```

Comment:
At least from what I can see in this diff, there is no subsequent reference to the value `v`, so there should be no reason to specify that name. This can be just "...from an on-chain value to a number..."